syft/__init__.py

```diff
@@ -1 +1,2 @@
 from .tensor import TensorBase
+from .tensor import *
```

Comment:
Yes, was hoping to get feedback on how this is normally implemented. Do you normally add every method to init?